manifests/install.pp

```diff
@@ -0,0 +1,44 @@
+# == Class nzbget::install
+#
+# This class is called from nzbget for install.
+#
+
+class nzbget::install {
+  include apt
+  if ($nzbget::manage_ppa) {
+    apt::ppa { 'ppa:modriscoll/nzbget': }
```

Comment:
NZBGet is now stored in official repositories. With that in mind, do we really need to add a ppa? If we think so, is this PPA reliable and official?